package.json

```diff
@@ -7,7 +7,12 @@
     "react-component-variations": "./bin.js"
   },
   "scripts": {
-    "lint": "eslint .",
+    "prebuild": "npm run clean",
+    "build": "babel src -d .",
+    "build:watch": "npm run build -- --watch",
+    "clean": "rm forEachVariation.js",
+    "lint": "eslint src",
```

Comment:
what about tests? i think this should be `eslint --ext=js,jsx .`, and then `build` added to `eslintignore`?